gatsby-config.mjs

```diff
@@ -97,6 +103,61 @@ const plugins = [
   }
 ];
 
+const algoliaPagesQuery = `
+{
+  pages: allFile(filter: {extension: {in: ["md", "mdx"]}}) {
+    nodes {
+      id
+      internal {
+        contentDigest
+      }
+      childMdx {
+        tableOfContents
+        fields {
+          slug
+        }
+        excerpt(pruneLength: 1000)
+        frontmatter {
+          title
+          description
+          toc
+          tags
+        }
+      }
+    }
+  }
+}
+`;
+
+const queries = [
+  {
+    query: algoliaPagesQuery,
+    transformer: ({ data }) => data.pages.nodes.map(node => {
+      const { childMdx, ...rest} = node;
+      delete rest.childMdx;
+      return {
+        ...childMdx,
+        ...rest
+      }
+    })
+  },
+];
+
+const algoliaPlugin = {
+  resolve: `gatsby-plugin-algolia`,
+  options: {
+    appId: process.env.GATSBY_ALGOLIA_APP_ID,
```

Comment:
```suggestion
    appId: process.env.ALGOLIA_APP_ID,
```

Reply:
forget :)

Reply:
I need that env variable in the browser and gatsby don't expose it if not prefixed with `GATSBY_`